### ISSUE TITLE
Not compatible with latest version of Resharper 2024.3

### ISSUE THREAD

**Issue by sjh37** [JetBrains/ForTea#77]
The ForTea extension is not available for Resharper 2024.3.5
Can this be compiled and released so it shows in the Extension manager.

![Image](https://github.com/user-attachments/assets/16bc6451-3d12-4a31-bd7f-371a742d3c50)

**Comment by aybe** [comment]
This is what happens as of today:

![image](https://github.com/JetBrains/ForTea/assets/1537591/4d14db33-753a-4f26-868f-7fbd1862db18)

Installation errors:

[Talisax.txt](https://github.com/user-attachments/files/15860635/Talisax.txt)

Log file:

[Sofapoh.txt](https://github.com/user-attachments/files/15860637/Sofapoh.txt)

**Comment by axunonb** [comment]
For ReSharper 2024.3.5 there is no compatible version of `ForTea`. Unfortunately we find this out only after installing the latest version, and then we have to go back. The [ForTea download page](https://plugins.jetbrains.com/plugin/13469-fortea/versions) showed 2024.3.5 as compatible.
It would be nice to have `ForTea` available together, or at least shortly after a new dotUltimate release. The current solution is painful.
